providers/ms365/resources/ms365.lr

```diff
@@ -448,6 +448,8 @@ ms365.teams {
   csTenantFederationConfiguration() ms365.teams.tenantFederationConfig
   // CS Teams meeting policy configuration
   csTeamsMeetingPolicy() ms365.teams.teamsMeetingPolicyConfig
+  // CS Teams message policy configuration
+  csTeamsMessagingPolicy() ms365.teams.teamsMessagingPolicyConfig
```

Comment:
We should probably consider not using the CS here since that goes back to the legacy Skype Communications Server (CS). The cmdlets in PowerShell use the names, but that doesn't mean we need to follow that same naming.

Reply:
removed CS, Thanks